com.unity.shadergraph/Editor/Data/Graphs/AbstractShaderProperty.cs

```diff
@@ -100,6 +100,11 @@ internal virtual void AppendPropertyBlockStrings(ShaderStringBuilder builder)
 
         internal abstract void ForeachHLSLProperty(Action<HLSLProperty> action);
 
+        internal virtual string GetPropertyAsArgumentString(string precisionString, bool isVfxGraph)
```

Comment:
If this is only ever called from VFX path, rename it to GetPropertyAsArgumentStringForVFX() and drop the bool.

Reply:
Renamed, thanks!